src/index.css

```diff
@@ -1,3 +1,9 @@
 #root {
   height: 100%;
 }
+
+input::selection,
+textarea::selection {
```

Comment:
Why not just `::selection`? If one day we'd have any contentEditable element it could also use that styling